### PR TITLE
Use OWNERS_ALIASES file

### PR DESCRIPTION
**Pull request by weinimo**
Be consistent with how other operators manage the OWNERS file.

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    - johnsom
+    - karelyatin
+    - slawqo
+    - weinimo
+  openstack-approvers:
+    - abays
+    - dprince
+    - gibizer
+    - olliewalsh
+    - stuggi
```
Comment (Sandeepyadav93) [inline OWNERS_ALIASES:19]:
~~~
fix end of files.........................................................Failed
- hook id: end-of-file-fixer
- exit code: 1
- files were modified by this hook
Fixing OWNERS_ALIASES
don't commit to branch..................................................Skipped
12 skipped lines...
     - gibizer
     - olliewalsh
-    - stuggi
\ No newline at end of file
+    - stuggi
~~~

Comment (Sandeepyadav93) [reply OWNERS_ALIASES:19]: johnsom and gthiemonge will need to join openstack-k8s-operators org to be part of OWNERS file, you can ping on #openstack-k8s-operators channel and someone can invite them to this group.

**Review by Sandeepyadav93 — CHANGES_REQUESTED** [review]

File: OWNERS_ALIASES (hunk 1 above)
Comment (weinimo) [reply OWNERS_ALIASES:19]: done already. thanks

**Comment by Sandeepyadav93** [comment]
/lgtm

octavia-operator-build-deploy-kuttl failure not related to this change.

**Pull request merged**